powerdns_sync/debian/atomiadns-powerdnssync.atomiadns-powerdnssync.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
```

Comment:
Is atomiadns-powerdnssync.atomiadns-powerdnssync.service the correct name for this or should it be atomiadns-powerdnssync.service ?